testsuite/MDAnalysisTests/topology/test_pdb.py

```diff
@@ -244,3 +248,26 @@ def test_PDB_metals():
     assert u.atoms[1].mass == pytest.approx(tables.masses["FE"])
     assert u.atoms[2].mass == pytest.approx(tables.masses["CA"])
     assert u.atoms[3].mass == pytest.approx(tables.masses["MG"])
+
+
+def test_PDB_elements():
+    from MDAnalysis.topology import tables
+
+    u = mda.Universe(PDB_elements, format='PDB')
+    element_list = ['N', 'C', 'C', 'O', 'C', 'C', 'O', 'N', 'H', 'H', 'H',
+                    'H','H','H', 'H', 'H', 'CU', 'FE', 'C', 'MG']
+    assert len(u.atoms.elements) == len(element_list)
+    assert all(i == j for i,j in zip(element_list, u.atoms.elements))
+
+
+
+def test_PDB_elements_warnings():
+    from MDAnalysis.topology import tables
```

Comment:
Why are you importing tables?

Reply:
The tables have the location for the PDB file with missing elements. But I would use the 'String IO' option instead as suggested in the next comment. That should resolve this